### PR TITLE
Build DataUri from remote URL

### PR DESCRIPTION
**Pull request by zanardigit**
* get file from a remote URL via CURL and build a DataUri from it
* perform basic CURL response validation

Already included in unit tests. I just wasn't sure which remote image to use from test, so I improvised :-)

### CODE REVIEW

**Comment by aztech-dev** [comment]
Thanks for the P/R. Do you mind adding ext-curl as a dev requirement in composer.json and updating the documentation accordingly ?

**Comment by zanardigit** [comment]
Thank you for your quick reply!
I've added the dependency to composer.
@aztech-dev isn't the documentation supposed to be generated via phpDoc? Is that what you're asking me to do?

**Comment by aztech-dev** [comment]
Great. For documentation I was refering to https://github.com/alchemy-fr/PHP-dataURI/blob/master/docs/source/index.rst , but I can handle it if you want.

**Comment by zanardigit** [comment]
Oh, sure :-) 
Done. I also fixed two typos in the doc just a few lines above, hope you don't mind.
Thank you.

**Comment by aztech-dev** [comment]
Thanks !

**Comment by zanardigit** [comment]
Eagerly waiting for a new release so I can issue a related PR here too: https://github.com/romainneutron/TwigExtension-DataUri 

:-)

**Comment by zanardigit** [comment]
I apologize in advance for bothering you again... is there anything I can do to speed up the release of a new version with this enhancement included?

**Comment by aztech-dev** [comment]
Done :)

**Comment by zanardigit** [comment]
Thank you so much!

**Comment by aztech-dev** [comment]
No problem. I meant to do it after your P/R but I got distracted..

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
   ],
   "require": {
     "php"                     : ">=5.3.2",
-    "symfony/http-foundation" : "~2.0"
+    "symfony/http-foundation" : "~2.0",
+    "ext-curl"                : "*"
```
Comment (romainneutron) [inline composer.json]: this should have been a soft dependency. Curl is not required to use buildFromFile.

Comment (aztech-dev) [inline composer.json]: https://github.com/alchemy-fr/PHP-dataURI/commit/aaacd921ae589d5072804eab1497230515925253#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R19 :)

**Pull request merged**